azurerm/resource_arm_role_assignment.go

```diff
@@ -7,6 +7,7 @@ import (
 
 	"github.com/Azure/azure-sdk-for-go/arm/authorization"
 	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/satori/uuid"
```

Comment:
Should we be using [go-uuid](https://github.com/hashicorp/go-uuid) here to stay in line with the rest of the HashiCorp ecosystem?

Reply:
possibly - the Azure SDK uses this version (and now `github.com/marstr/guid`) - so I picked it for internal consistency tbh; I'll update this